feature-libs/organization/administration/components/unit/details/unit-details.component.ts

```diff
@@ -30,5 +31,10 @@ export class UnitDetailsComponent {
   );
   isInEditMode$ = this.itemService.isInEditMode$;
 
-  constructor(protected itemService: ItemService<B2BUnit>) {}
+  isUpdatingUnitAllowed = this.orgUnitService?.isUpdatingUnitAllowed();
```

Comment:
we can use `readonly` to prevent modification after assignment

Reply:
done